Sources/MongoSwift/MongoCollection+Indexes.swift

```diff
@@ -48,7 +48,7 @@ public struct IndexOptions: Encodable {
      *
      * - Example: For an index of name: 1, age: -1, the generated name would be "name_1_age_-1".
      */
-    public let name: String?
+    public var name: String?
```

Comment:
this and `indexVersion` became mutable for the sake of the test. I considered updating all of them but I think that ticket will make it into our next release anyway